spec/system/reports_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reports', type: :system do
+  before do
+    user = User.create!(name: 'alice', email: 'alice@example.com', password: '123456', password_confirmation: '123456')
+    @report = Report.create(user_id: user.id, content: 'content', title: 'title')
```

Comment:
```suggestion
    @report = user.reports.create!(content: 'content', title: 'title')
```

のほうがいいですね。
あと、contetやtitleはリアルなデータで！